.github/workflows/pull-request.yml

```diff
@@ -39,7 +39,12 @@ jobs:
         env:
           TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
           TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
-          NEXT_TELEMETRY_DISABLED: 1
+
+      - name: Run Prettier
+        run: npx turbo prettier
+        env:
+          TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+          TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
```

Comment:
@ovflowd @araujogui  I have figured out why this Job is failing. 

We're running the lint stage before, which causes the `npm run lint:js` to execute, which modified the empty `blog-post-data.json` with data in the form of minified json. So when we run the prettier stage after, that file isn't in the pretty JSON format, and therefore this CI fails.

Easy fix:
Reorder the above stages to run Prettier before Linting.

Reply:
But shouldn’t prettier and eslint not conflict with each other?

Reply:
@ovflowd, They shouldn't, but in this case we are causing a conflict because we're create a file at the lint check stage using `generateBlogPostsData.mjs`. I don't even know why we're doing that.

Reply:
Why is generateBlogData being… Ah, because of “next lint” command?

We might need to introduce a check on next.config.mjs…

Or we should make a “prebuild” script to actually run the stuff we use for file generation.

Do you follow me? Can you make a quick PR? Probably a “prebuild” package.json step is easier.

Then we remove the lines of generating file from next.config to a dedicate file for these file generation. Like next.data.mjs

Reply:
Once we commit any thing, the pre commit hooks run, which reset `blog-post-data.json` to `{ "pagination": [], "categories": [], "posts": [] }`, so there's nothing wrong in the commit, and all the files are formatted correctly.

If we run prettier before, it would always check `{ "pagination": [], "categories": [], "posts": [] }` and pass.

Reply:
After running Lint, the file looks more like this
<img width="699" alt="image" src="https://github.com/nodejs/nodejs.org/assets/10580591/a2fdf29a-8a92-464b-9513-ef9f71149ceb">

Reply:
Read my last comment 👀

Reply:
Another easy workaround would be to just ignore this file.

Reply:
I think we need to fix regardless of this the next.config issue 😅

Reply:
@ovflowd, I'm sorry something went wrong with my GitHub and didn't show your comments until now. I agree with what you propose.